data/json/items/vehicle/cables.json

```diff
@@ -23,7 +23,7 @@
     "type": "TOOL",
     "id": "extension_cable",
     "name": { "str": "extension cord" },
-    "description": "A long 10m orange extension cord for connecting appliances.  Could be used on any appliance or other household electrical system.",
+    "description": "A long 30-ft (or about 9 m) orange extension cord for connecting appliances.  Could be used on any appliance or other household electrical system.",
```

Comment:
actual length and description length no longer line up for this one.

Reply:
since the tow cable is now seriously wrong. The charges are the number of meters the cable extends.

Reply:
Ik the tow cable is weird. This was the test setup with this being 1 tile apart, went snap on 24 tiles between the bumpers.
![Screenshot from 2022-03-16 20-14-24](https://user-images.githubusercontent.com/93329733/158712403-c1460920-0d9b-48aa-abc7-f6ead878061d.png) (the 2 white # is solid rock)
.